libvmi/driver/kvm/kvm.c

```diff
@@ -70,12 +70,18 @@ exec_qmp_cmd(
     char *output = safe_malloc(20000);
     size_t length = 0;
 
-    char *name = (char *) virDomainGetName(kvm->dom);
+    const char *name = virDomainGetName(kvm->dom);
     int cmd_length = strlen(name) + strlen(query) + 29;
     char *cmd = safe_malloc(cmd_length);
 
-    snprintf(cmd, cmd_length, "virsh qemu-monitor-command %s %s", name,
+    int rc = 0;
+
+    rc = snprintf(cmd, cmd_length, "virsh qemu-monitor-command %s %s", name,
```

Comment:
No point initializing rc to zero just to be overwritten with snprintf right after.